backend/src/main/java/com/huellapositiva/domain/service/ProposalService.java

```diff
@@ -107,4 +110,23 @@ public ProposalRevisionEmail requestChanges(
     private boolean hasFeedback(ProposalRevisionDto proposalRevisionDto) {
         return proposalRevisionDto.getFeedback() != null;
     }
+
+    /**
+     * This method find the proposal in the database and checks if the status is PUBLISHED for
+     * close the enrollment. Otherwise, a ProposalNotClosableException with response status 409 will be throw.
```

Comment:
Too many technical details. You don't need to explain the code below in the Javadoc.

```
Close the enrollment of the provided proposal.
@param 
@throws XXXException when the proposal is not published.
```